test/server.jl

```diff
@@ -0,0 +1,24 @@
+# find a good port
+conn = listenany(2228)
+close(conn[2])
+port = conn[1]
+
+@async lintserver(port)
+sleep(1) #let sever start
```

Comment:
@omus It has to be in @async because the server runs forever. The reason I sleep is because accept is the 4th line in lintserver and it has to run before connect on the next line of the test.